app/models/user.rb

```diff
@@ -344,7 +344,18 @@ def destroy
     super
   end
 
-  %w[military_information payment].each do |emis_method|
+  military_information_method = 'military_information'
+  define_method(military_information_method) do
```

Comment:
Please get rid of this meta programming, it doesn't serve a purpose and is really confusing. These methods are simple enough that there's little risk in just unraveling the meta programming (I know you all weren't the ones who introduced it but it'll make everyone's lives easier if you just unravel it right now)

Reply:
Will do! 👍

Reply:
@bosawt ready for another look!